### PR TITLE
feature(client-relational): cleaner interfaces

### PR DESCRIPTION
**Pull request by rakibansary**

### CODE REVIEW

File: clients/client-relational/src/interfaces/Schema.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { TableColumns } from "./TableColumns";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (hamidre13) [inline clients/client-relational/src/interfaces/Schema.ts:3]: no needed based on the rule

File: clients/client-relational/src/query/BaseQuery.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Schema } from "../interfaces/Schema";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (hamidre13) [inline clients/client-relational/src/query/BaseQuery.ts:3]: No needed

File: clients/client-relational/src/query/QueryBuilder.ts [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+}
+
+interface Select {
+  where: (column: TableColumn, operator: Operator, value: Value) => Where;
+  limit: (limit: number) => Limit;
+  offset: (offset: number) => Build;
+  build: () => Query;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (hamidre13) [inline clients/client-relational/src/query/QueryBuilder.ts:34]: same

**Review by hamidre13 — COMMENTED** [review]
I left some style comments

File: clients/client-relational/src/interfaces/Schema.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+export type Schema<T extends Record<string, any>> = {
```
[the remark this answers is not in the document]
Comment (rakibansary) [reply clients/client-relational/src/interfaces/Schema.ts:4]: Schema here is in the context of a relational database and should be clear from @topcoder-framework/client-relational usage

**Pull request merged**